components/image/inkscape/Makefile

```diff
@@ -0,0 +1,91 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2015 Aurelien Larcher
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		inkscape
+COMPONENT_VERSION=	0.91
+COMPONENT_FMRI=		image/editor/inkscape
+COMPONENT_CLASSIFICATION=	Applications/Graphics and Imaging
+COMPONENT_SUMMARY=	Opensource professional vector graphics editor
+COMPONENT_PROJECT_URL=	http://www.inkscape.org/
+COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.bz2
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:4d901f8a9e1924404e797ad23b8b0c495a9d155448816d95a55974314e1f141b
+COMPONENT_ARCHIVE_URL=	http://inkscape.org/en/gallery/item/3860/$(COMPONENT_ARCHIVE)
+COMPONENT_LICENSE = GPLv2
+COMPONENT_LICENSE_FILE = $(COMPONENT_NAME).license
```

Comment:
Just a little hint - I made COMPONENT_LICENSE_FILE default to $(COMPONENT_NAME).license in https://github.com/OpenIndiana/oi-userland/commit/fd5ab52f5aebb38d382ee662954fb9e82e1da98a, so if you don't change it, you don't have to add that line. I fine either way,